registry-library/library/library_test.go

```diff
@@ -709,3 +712,283 @@ func TestDownloadStarterProjectAsBytes(t *testing.T) {
 		})
 	}
 }
+
+func TestDownloadStarterProject(t *testing.T) {
+	close, err := setUpTestServer(func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think this is the same inner function that's in `TestDownloadStarterProjectAsDir`.  Is there a way to refactor this to avoid duplication?

Reply:
@kim-tsao Simplified it down to a single handler.